feature-libs/product/configurators/common/src/components/config-group-menu/config-group-menu.component.ts

```diff
@@ -93,13 +100,18 @@ export class ConfigGroupMenuComponent implements OnInit {
     });
   }
 
-  navigateUpOnEnter(event) {
+  /**
+   * Navigates to the subgroup.
+   *
+   * @param event
+   */
+  navigateUpOnEnter(event): void {
```

Comment:
Is there a type for this `event` that we can use?

Reply:
is **any** as a type okay?

Reply:
Please see https://github.com/SAP/spartacus/pull/8434#discussion_r469107010